### PR TITLE
feat: styled footer links

### PR DESCRIPTION
**Pull request by vm**
Screen Shot 2022-12-11 at 11 25 18 PM
Screen Shot 2022-12-11 at 11 25 23 PM
Screen Shot 2022-12-11 at 11 25 41 PM

### CODE REVIEW

File: src/pages/About/index.tsx [hunk 1]
```diff
@@ -27,7 +28,7 @@ const PageBackground = styled.div<{ isDarkMode: boolean }>`
   position: absolute;
   width: 100%;
   height: 100vh;
-  top: 0;
+  top: -72px;
```
Comment (grabbou) [inline src/pages/About/index.tsx:31]: I think you can replace this with a navHeight from the theme?

File: src/pages/About/index.tsx [hunk 2]
```diff
@@ -134,12 +135,42 @@ const Thumbnail = styled.img`
...
+  }
+`
+
+const FooterLink = styled.a`
+  display: flex;
+  align-items: center;
+  gap: 16px;
+  padding: 16px;
+  color: ${({ theme }) => theme.textPrimary};
+  border: 1px solid ${({ theme }) => theme.backgroundOutline};
```
Comment (grabbou) [inline src/pages/About/index.tsx:158]: Have you tried using a ButtonOutline or similar style from the existing collection of components?

**Review by grabbou — APPROVED** [review]
Overall looks good, left two nits

File: src/pages/About/index.tsx (hunk 1 above)
Comment (vm) [reply src/pages/About/index.tsx:31]: yes

**Pull request merged**